PocketKit/Sources/SharedPocketKit/Session/AppSession.swift

```diff
@@ -1,10 +1,21 @@
 import Foundation
 
+public enum UserStatus {
+    case premium
+    case free
+}
+
 public class AppSession {
     @KeychainStorage
     public var currentSession: Session?
+    public var currentStatus: UserStatus?
```

Comment:
Can we just default to `free` and update as necessary?

Reply:
I wanted to `nil` it out when the user is logged out, but I can have it default to `free`

Reply:
@cyndichin makes sense; similar thought process to `currentSession` then. an alternative is adding another `loggedOut` case, but I'll leave that decision up to you.

Reply:
I decided to make an `unknown` case just since the user may be logged in but the Fetch Saves query was not completed.